docs/glossary.md

```diff
@@ -11,74 +6,116 @@ sidebar_position: 27
 
 # Glossary
 
+List of frequently used terms and acronyms.
+
 <a id="term-ABI"></a>
 
 ## ABI
 
-**A**pplication **B**inary **I**nterface. ABI is a document that comprehensively defines the binary system interface between applications and the operating system on which they run. [Learn More at Wikipedia](https://en.wikipedia.org/wiki/Application_binary_interface) or [pypackaging-native](https://pypackaging-native.github.io/background/binary_interface).
```

Comment:
Why did you remove the `pypackaging` reference?

Reply:
Snafu. It's back now.